sematic/resolvers/resource_managers/in_memory_manager.py

```diff
@@ -3,7 +3,7 @@
 from typing import Dict, FrozenSet, List, Tuple
```

Comment:
Should this class be called `MemoryResourceManager` to align with `MemoryStorage` and `CloudResourceManager`?

Reply:
done

Reply:
changed